src/pages/Content/CodeGen.tsx

```diff
@@ -2,196 +2,10 @@ import React from 'react';
 import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
```

Comment:
One of the main goal is to move all code gen logic out of `CodeGen.tsx` component